packages/theme/src/theme.ts

```diff
@@ -41,6 +41,26 @@ export type InteractiveColor = {
   destructive: ContrastColor;
 };
 
+export enum GeofencingZoneCodes {
+  Allowed = 'Allowed',
+  Slow = 'Slow',
+  NoParking = 'NoParking',
+  NoEntry = 'NoEntry',
+}
+
+export type GeofencingZoneKeys = keyof typeof GeofencingZoneCodes;
+
+export type GeofencingZonePaintProps = {
```

Comment:
I got a bit hung up in the naming here 🤓 I'm ok with verbose type names when needed but 

- `GeofencingZone`: What's the difference between a "GeofencingZone" and a "Geofence"? (I realize that changing it might affect more than this design system change, so I can understand wanting to keep it consistent)
- `Paint`: Is that relevant for a type from the design system package?
- `Props` sounds like it's specific to one component i React. I don't think the design system should be that implementation specific.

Is `GeofenceStyle` just as precise a name?

Reply:
The `geofencingZone` naming comes [from GBFS](https://github.com/MobilityData/gbfs/blob/v2.2/gbfs.md#geofencing_zonesjson-added-in-v21) and is used consistently in the frontend implementation, so I think we should keep that.
However if you prefer we can change it to `GeofencingZoneStyle` instead?

Reply:
Update: https://github.com/AtB-AS/design-system/pull/174/commits/9d27bf6f2b96a1337971929a7e0578957f0ff9b7